python/nav/mibs/pdu2_mib.py

```diff
@@ -0,0 +1,181 @@
+#
+# Copyright (C) 2017 UNINETT
+#
+# This file is part of Network Administration Visualized (NAV).
+#
+# NAV is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License version 2 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.  You should have received a copy of the GNU General Public License
+# along with NAV. If not, see <http://www.gnu.org/licenses/>.
+#
+"""A class for getting sensor information from Raritan PDU2 devices
+"""
+
+from twisted.internet import defer
+from twisted.internet.defer import returnValue
+from nav.mibs.mibretriever import MibRetriever
+
+UNIT_MAP = {
+    'none': '',
+    'other': 'other',
+    'volt': 'volt',
+    'amp': 'amperes',
+    'watt': 'watt',
+    'voltamp': 'voltamperes',
+    'wattHour': 'watthour',
+    'voltampHour': 'voltamperehour',
+    'degreeC': 'Celsius',
+    'hertz': 'hertz',
+    'percent': '%',
+    'meterpersec': 'meter/second',
+    'pascal': 'pascal',
+    'psi': 'psi',
+    'g': 'grams',
+    'degreeF': 'Fahrenheit',
+    'feet': 'feet',
+    'inches': 'inches',
+    'cm': 'meters',
+    'meters': 'meters',
+    'rpm': 'rpm',
+    'degrees': 'degrees',
```

Comment:
I realize the name comes from the MIB, but does it refer to angles or temperatures? :D